frontend/src/metabase/entities/users.js

```diff
@@ -1,28 +1,28 @@
 import { assocIn } from "icepick";
 
+import { userApi, sessionApi } from "metabase/api";
 import * as MetabaseAnalytics from "metabase/lib/analytics";
-import { GET } from "metabase/lib/api";
-import { createEntity } from "metabase/lib/entities";
+import { createEntity, entityCompatibleQuery } from "metabase/lib/entities";
 import { generatePassword } from "metabase/lib/security";
 import MetabaseSettings from "metabase/lib/settings";
 import { UserSchema } from "metabase/schema";
-import { UserApi, SessionApi } from "metabase/services";
 
 export const DEACTIVATE = "metabase/entities/users/DEACTIVATE";
 export const REACTIVATE = "metabase/entities/users/REACTIVATE";
 export const PASSWORD_RESET_EMAIL =
   "metabase/entities/users/PASSWORD_RESET_EMAIL";
 export const PASSWORD_RESET_MANUAL =
   "metabase/entities/users/RESET_PASSWORD_MANUAL";
-export const RESEND_INVITE = "metabase/entities/users/RESEND_INVITE";
 
 // TODO: It'd be nice to import loadMemberships, but we need to resolve a circular dependency
 function loadMemberships() {
   return require("metabase/admin/people/people").loadMemberships();
 }
 
-const getUserList = GET("/api/user");
-const getRecipientsList = GET("/api/user/recipients");
+const getUserList = (query = {}, dispatch) =>
+  entityCompatibleQuery(query, dispatch, userApi.endpoints.listUsers);
+const getRecipientsList = dispatch =>
+  entityCompatibleQuery({}, dispatch, userApi.endpoints.listUserRecipients);
```

Comment:
Maybe it would've been cleaner to pass both the `query` and the `dispatch` to both functions?

Reply:
https://github.com/metabase/metabase/pull/41312/commits/d09565283d505039356bc19fbcf1cc10cb49b933